tutorials/parsl-rp/parsl-rp.ipynb

```diff
@@ -0,0 +1,339 @@
+    "rpex_cfg = 'configs/rpex.cfg'\n",
+    "config = Config(\n",
+    "    executors=[RadicalPilotExecutor(\n",
+    "        rpex_cfg=rpex_cfg, bulk_mode=True,\n",
+    "        resource='local.localhost', login_method = 'local',\n",
+    "        walltime=30, managed= True, cores= 4\n",
+    "    )])\n",
+    "\n",
+    "parsl.load(config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a6f2135",
+   "metadata": {},
+   "source": [
+    "Create a simple Parsl `@bash_app` to invoke the `NWChem` task. The `bash_app` requires the type of the task and the number of `cpu_processes` on which to run. In this case, the type of the task is `MPI`, and the number of `cpu_processes` is 2 `MPI` ranks, where each rank takes 1 core.\n",
+    "\n",
+    "Once the `bash_app` (executable task) is invoked, the RadicalPilotExecutor submits the task to the runtime system and wait for them to be executed. RadicalPilotExecutor creates a designated `sandbox` folder that contains the tasks and their `stdout/stderr` files."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad3e90a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@parsl.bash_app\n",
+    "def nwchem_mp2_optimization(cpu_processes=2, cpu_process_type=rp.MPI):\n",
+    "\n",
+    "    return '{0} {1}'.format(nwchem, nwchem_input)\n",
+    "\n",
+    "# invoke the nwchem_mp2_optimization\n",
+    "future  = nwchem_mp2_optimization()\n",
+    "\n",
+    "# wait for the results of the NWChem task.\n",
+    "if future.result() == 0:\n",
+    "    print('Parsl task {0} finished'.format(future.tid))\n",
+    "    \n",
+    "    # rp has a different task id than Parsl (task.id)\n",
+    "    task_id  = str(future.tid).zfill(6)\n",
+    "\n",
+    "    # RP tasks output located in the sandbox folder\n",
+    "    task_path = '{0}/radical.pilot.sandbox/{1}/pilot.0000/task.{2}/task.{2}.out'.format(os.path.expanduser('~'),\n",
+    "                                                                                        config.executors[0].session.uid, task_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "34d6ebb7",
+   "metadata": {},
+   "source": [
+    "print task output from the task file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fa510510",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "task_out = open(task_path, 'r').readlines()\n",
+    "print(''.join(task_out))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e198583e",
+   "metadata": {},
+   "source": [
+    "Finally, shutdown the executor, otherwise it will always stays ready to get more tasks"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e9851cb1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config.executors[0].shutdown()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
```

Comment:
@mturilli just a side comment, if you can manually update this part to the following one, then tutorial will be opened within the environment without confirming it
```
  "kernelspec": {
    "display_name": "Python [conda env:rct]",
    "language": "python",
    "name": "conda-env-rct-py"
   },
```